tee-worker/litentry/core/credentials/src/credentials.rs

```diff
@@ -0,0 +1,269 @@
+// Copyright 2020-2022 Litentry Technologies GmbH.
```

Comment:
The file structure `credentials/src/credentials.rs` seems strange: we have a crate and file with the same name. Can we move the content of this file to `lib.rs`?

Reply:
for sure, will refactor this model and add comments.